src/types/bops.ts

```diff
@@ -33,10 +12,10 @@ interface BOPSMinimumPayload {
     address_2?: string;
     town?: string;
     postcode?: string;
-    latitude: string;
-    longitude: string;
-    x: string;
-    y: string;
+    latitude: number;
+    longitude: number;
+    x: number;
+    y: number;
```

Comment:
Follow on from dev call - these are all currently sent to BoPS as `number` so this is correct. I checked a few recent records on `bops_applications`.